### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by apigonspyro** [visgl/flowmap.gl#19] [Q&A] [answered]
Hi I was using @flowmap.gl/react package and recently tried upgrading react-map-gl to v7, after that I'm getting error that @flowmap.gl/react is using old api of react-map-gl. I've tried upgrading @flowmap.gl/react package as well but noticed that it is no longer supported and is being migrated to this repo. Could you advise how to install and use new package or when it will be avaiable?

**Comment by ilyabo** [comment, marked as answer]
You should upgrade to the new version of the library v8.  Check the [examples here](https://github.com/FlowmapBlue/flowmap.gl/tree/main/examples)
